store/tikv/error.go

```diff
@@ -22,6 +22,8 @@ import (
 var (
 	// ErrBodyMissing response body is missing error
 	ErrBodyMissing = errors.New("response body is missing")
+	// ErrMismatchClusterID returns the error that the cluster ID of the PD client does not match the PD.
+	ErrMismatchClusterID = errors.New("mismatch cluster id")
```

Comment:
We don't need to define an error if we never throw it.
I think just make the string constant and check the string will do.

Reply:
Agree